package/gluon-autoupdater/src/respondd.c

```diff
@@ -1,27 +1,5 @@
-/*
-  Copyright (c) 2016, Matthias Schiffer <mschiffer@universe-factory.net>
-  All rights reserved.
-
-  Redistribution and use in source and binary forms, with or without
-  modification, are permitted provided that the following conditions are met:
-
-    1. Redistributions of source code must retain the above copyright notice,
-       this list of conditions and the following disclaimer.
-    2. Redistributions in binary form must reproduce the above copyright notice,
-       this list of conditions and the following disclaimer in the documentation
-       and/or other materials provided with the distribution.
-
-  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
-  AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
-  IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
-  DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
-  FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
-  DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
-  SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
-  CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
-  OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
-  OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
-*/
+// SPDX-FileCopyrightText: 2016, Matthias Schiffer <mschiffer@universe-factory.net>
+// SPDX-License-Identifier: BSD-2-Clause
```

Comment:
I'd prefer C-style comments

Reply:
```suggestion
/*
SPDX-FileCopyrightText: 2016, Matthias Schiffer <mschiffer@universe-factory.net>
SPDX-License-Identifier: BSD-2-Clause
*/
```
Like this?

```suggestion
/* SPDX-FileCopyrightText: 2016, Matthias Schiffer <mschiffer@universe-factory.net> */
/* SPDX-License-Identifier: BSD-2-Clause */
```
or this?

Reply:
This the editorconfig obviously does not allow:

```suggestion
/* SPDX-FileCopyrightText: 2016, Matthias Schiffer <mschiffer@universe-factory.net>
   SPDX-License-Identifier: BSD-2-Clause */
```

This it would though:

```suggestion
/* SPDX-FileCopyrightText: 2016, Matthias Schiffer <mschiffer@universe-factory.net>
 * SPDX-License-Identifier: BSD-2-Clause */
```

So the most similar to before (and my favorite after c++ style comments) would be this?

```suggestion
/*
 * SPDX-FileCopyrightText: 2016, Matthias Schiffer <mschiffer@universe-factory.net>
 * SPDX-License-Identifier: BSD-2-Clause
 */
```

Reply:
Blocktrron favors the second variant.

*/ then that it is */

Reply:
resolved in d374d6dfcf062d4d0f18352926e391a85de7de78